### PR TITLE
[recharts] adjusted 'utcTime' to 'utc' according to rechats.js

### PR DESCRIPTION
**Pull request by t-knapp**
Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/recharts/recharts/issues/1195
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Review by G07cha — APPROVED** [review]

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## recharts ([<kbd>unpkg</kbd>](https://unpkg.com/browse/recharts@latest/))
was missing the following properties:
1. Layer
1. Sankey
1. Funnel
1. FunnelChart
1. Trapezoid
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 63841e0eadf74442c3c26bfb1b13a8f09a08470f
</p>

**Review by dimitropoulos — APPROVED** [review]

**Review by gndelia — APPROVED** [review]

**Comment by orta** [comment]
This has merge conflicts on adding yourself as a maintainer - if you reabse we can merge it 👍

File: types/knuddels-userapps-api/index.d.ts [hunk 1]
```diff
@@ -11,2981 +11,3075 @@ export type KnuddelsEvent = string | Json | KnuddelsEventArray;
 
 // helper interfaces
 declare global {
-    interface Json {
-        [x: string]: JsonData | undefined;
-    }
+	interface Json {
```
Comment (danger-public) [inline types/knuddels-userapps-api/index.d.ts:14]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Review by johnnyreilly — APPROVED** [review]

**Comment by t-knapp** [comment]
Ready to merge

**Pull request merged**